node/src/components/block_synchronizer/block_builder/tests.rs

```diff
@@ -25,38 +25,38 @@ fn handle_acceptance() {
 
     // Builder acceptance for needed signature from ourselves.
     assert!(builder
-        .handle_acceptance(None, Ok(Some(Acceptance::NeededIt)))
+        .handle_acceptance(None, Ok(Some(Acceptance::NeededIt)), true)
```

Comment:
Can you add a single case to this test when the `should_unlatch` is `false`?

Reply:
Added a new test in https://github.com/casper-network/casper-node/pull/4205/commits/896653b1abdade591bd96cc092faa4199e98fbd0 and renamed the old one for more granular testing.